packages/feedback/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { TestBed, async } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AppComponent } from './app.component';
+
+describe('AppComponent', () => {
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        RouterTestingModule
+      ],
+      declarations: [
+        AppComponent
+      ],
+    }).compileComponents();
+  }));
+
+  it('should create the app', () => {
+    const fixture = TestBed.createComponent(AppComponent);
```

Comment:
You can add this redundant lines of code to a separate `beforeEach`

Reply:
i tried it removing but i am getting error "cant find app or fixture"
Please check bellow code.
http://pastebin.test.redhat.com/849736

Reply:
Done.